### PR TITLE
Show blank votes & alternatives with 0 votes in election summary

### LINKED ISSUE #395: Should be able to show how many "blank" vote there are
Issue found on test day. One should be able to see how many voted blank. This is important for some reason I don't remember, but let's just put it in 👍🏻

### LINKED ISSUE #400: STV should report alternatives that got zero votes
With the current implementation, an alternative that gets 0 votes will not be listed in the iterations.

Example below. Here the alternative "Smiht" is not listed in the log, but it was one of the alternatives. It got none of the "first places" votes and is therefore removed from the iteration log. This is kinda unfortunate, as it can seem like a bug that it didn't show, or that it wasn't counted at all.
image

### PR DESCRIPTION
**Pull request by LudvigHz**
Fixes #395 
Fixes #400 

image

Also increases font size for the log and adds some tooltips.

### CODE REVIEW

File: app/stv/stv.ts [hunk 1]
```diff
@@ -178,7 +194,7 @@ exports.calculateWinnerUsingSTV = (
       // that have either won or been eliminated from the election.
       const currentAlternative = cloneDeep(vote.priorities[0]);
 
-      // Use the alternatives description as key in the counts, and add one for each count
+      // Use the alternatives description as key in the eounts, and add one for each count
```
Comment (SmithPeder) [inline app/stv/stv.ts:197]: Thanks for nothing 🙃

**Review by SmithPeder — APPROVED** [review]
Looks good, it would nice if you added some new tests for this. I don't think any of the old datasets has any blank votes, and/or alternatives that never got any votes. It would be nice to cover these cases.

**Comment by LudvigHz** [comment]
> Looks good, it would nice if you added some new tests for this. I don't think any of the old datasets has any blank votes, and/or alternatives that never got any votes. It would be nice to cover these cases. 

Yeah. I was planning on adding some tests, but haven't gotten round to it just yet 😅

**Review by SmithPeder — APPROVED** [review]
Nice

**Pull request merged**